monkey/monkey_island/cc/repository/i_event_repository.py

```diff
@@ -0,0 +1,35 @@
+from abc import ABC, abstractmethod
+from typing import Sequence
+
+from common.events import AbstractAgentEvent
+
+
+class IEventRepository(ABC):
```

Comment:
We need the ability to get events by type, tag, or source. Add 3 methods for those.